test/integration/shared_vpc_no_subnets/controls/gcloud.rb

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+group_email                 = attribute('group_email')
+project_id                  = attribute('project_id')
+project_number              = attribute('project_number')
+service_account_email       = attribute('service_account_email')
+shared_vpc                  = attribute('shared_vpc')
+
+control 'project-factory-shared-vpc' do
+  title "Project Factory shared VPC"
+
+  only_if { !(shared_vpc.nil? || shared_vpc == '') }
```

Comment:
Cleanup/nonblocker: this `only_if` expression stems from when I wrote a single integration test suite that ran with different parameters to test different modes. Since this test requires a shared VPC I think we can drop this check.

It may be better to follow up on this in a later PR, so if we want a quick turnaround resolve this issue and ping me; I'll file a ticket to follow up on this.